charts/renku/values.yaml

```diff
@@ -131,8 +131,12 @@ keycloak:
   ## keycloak admin username/password is supplied.
   initRealm:
     image:
-      repository: renku/init-realm
-      tag: 'latest'
+      repository: degano/init-realm
```

Comment:
Same here, I guess you'd clean this and the tests images up before merging as well.
We can publish the official tests image after merging.

Reply:
Yes, indeed.